README.md

```diff
@@ -160,6 +166,13 @@ afctl config show
 
 * Prints the configuration file on the console.
 
+### Example
+* Qubole
+```bash
+afctl config add -d qubole -n name -e https://api.qubole.com -c cluster-label -t Auth-token
+afctl deploy qubole -n name
```

Comment:
It should be `afctl config add -d qubole -n <name> -e <qubole env url> -c <cluster-label> -t <auth-token>`